src/frontend/templates/pages/networks.html

```diff
@@ -1,57 +1,61 @@
 {% extends 'base.html' %}
 
-{% block title %}Bulma Site{% endblock %}
+{% block title %}
+{% if not run %}
```

Comment:
Thanks. My emacs doesn't auto-indent because it recognizes html but it doesn't recognize django template language. I just go with the indention my emacs puts in automatically because it is easier (and doing differently requires me to fight the editor, which is not fun).

I think the block structure is hopefully not too bad because I use additional blank lines in places to try to give visual separation when appropriate.